_maps/shuttles/emergency_stormrider.dmm

```diff
@@ -0,0 +1,66 @@
+"a" = (/obj/structure/lattice,/obj/machinery/satellite/meteor_shield{active = 1},/turf/open/space/basic,/area/shuttle/escape)
```

Comment:
Make a subtype of meteor_shield that toogles on Initialize() instead, so it can update icon, anchored status and its floaty animation.